spec/support/product_file_list_helpers.rb

```diff
@@ -5,7 +5,7 @@
 module ProductFileListHelpers
   def have_file_row(name:, count: nil)
     options = { text: name, exact_text: true, count: }.compact
-    have_selector("[aria-label=Files] [role=treeitem] .content h4", **options)
+    have_selector("[aria-label=Files] [role=treeitem] h4", **options)
```

Comment:
I don't think `.content` is important to include; we weren't using it in some matchers in this file anyway